schema205/render_template.py

```diff
@@ -199,7 +200,24 @@ def add_table(
                 target,
                 caption=caption,
                 add_training_ws=False)
-    return add_table
+    return add_schema_table
+
+def make_add_yaml_table():
+    """
+    RETURN:
+    """
```

Comment:
Just a heads-up on the empty doc string but not a big deal.

Reply:
Fixed in the latest commit.